client/components/mma/accountoverview/manageProducts/ManageProductV2.tsx

```diff
@@ -244,31 +257,23 @@ const InnerContent = ({
 								Update payment method
 							</LinkButton>
 						)}
-
-					<div
-						css={css`
-							margin-left: ${space[5]}px;
-						`}
-					>
-						{!hasCancellationPending && (
-							<Button
-								priority="subdued"
-								onClick={() => {
-									navigate(
-										'/cancel/' +
-											specificProductType.urlPart,
-										{
-											state: {
-												productDetail: productDetail,
-											},
-										},
-									);
-								}}
+					{!hasCancellationPending && !isSelfServeCancellationAllowed && (
+						<div
+							css={css`
+								display: flex;
+								align-items: center;
+								justify-content: center;
+							`}
+						>
+							<Link
```

Comment:
I'm not sure I understand - the previous style seemed fine to me. And we should probably rather use the source Button than a react-router-link. What do you think?

Reply:
Apologies. I misunderstood the requirements of this ticket (and novice confusion with the switch system didn't help). Previous style now reinstated.

Reply:
no need to apologise! sorry there wasn't a clear requirement